docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -0,0 +1,459 @@
+  is the bucket that will host all the data written in this region;
+- :code:`awsEndpoint` : set to your bucket's endpoint, usually :code:`s3.amazonaws.com`;
+- :code:`bucketMatch` : set to :code:`true` if you want your object name to be the
+  same in your local bucket and your AWS S3 bucket; set to :code:`false` if you
+  want your object name to be of the form :code:`{{localBucketName}}/{{objectname}}`
+  in your AWS S3 hosted bucket;
+- :code:`credentialsProfile` and :code:`credentials` are two ways to provide
+  your AWS S3 credentials for that bucket, *use only one of them* :
+  - :code:`credentialsProfile` : set to the profile name allowing you to access
+    your AWS S3 bucket from your :code:`~/.aws/credentials` file;
+  - :code:`credentials` : set the two fields inside the object (:code:`accessKey`
+    and :code:`secretKey`) to their respective values from your AWS credentials.
+
+.. code:: json
+
+    (...)
+    "aws-test": {
+        "type": "aws_s3",
+        "legacyAwsBehavior": true,
+        "details": {
+            "awsEndpoint": "s3.amazonaws.com",
+            "bucketName": "zenkobucket",
+            "bucketMatch": true,
+            "credentialsProfile": "zenko"
+        }
+    },
+    (...)
+
+.. code:: json
+
+    (...)
+    "aws-test": {
+        "type": "aws_s3",
+        "legacyAwsBehavior": true,
+        "details": {
+            "awsEndpoint": "s3.amazonaws.com",
+            "bucketName": "zenkobucket",
+            "bucketMatch": true,
+            "credentials": {
+                "accessKey": "WHDBFKILOSDDVF78NPMQ",
+                "secretKey": "87hdfGCvDS+YYzefKLnjjZEYstOIuIjs/2X72eET"
+            }
+        }
+    },
+    (...)
+
+.. WARNING::
+   If you set :code:`bucketMatch` to :code:`true`, we strongly advise for having
+   only one local bucket per AWS S3 location. Indeed, since your objects names
+   won't be prefixed by the local bucket name in the AWS S3 bucket, you could
+   create inconsistencies seamlessly by putting two objects with the same name
+   in two different local buckets, but since they are ultimately saved in the
+   same AWS bucket, the most recent one would overwrite the earlier one, as the
+   namespace will conflict.
+
+config.json
+^^^^^^^^^^^
+
+Edit the :code:`restEndpoint` section of your :code:`config.json` file to add
+an endpoint definition matching your new AWS S3 hosted region. Following our
+previous example, it would look like:
+
+.. code:: json
+
+    (...)
+        "restEndpoints": {
+        "localhost": "us-east-1",
+        "127.0.0.1": "us-east-1",
+        "cloudserver-front": "us-east-1",
+        "s3.docker.test": "us-east-1",
+        "127.0.0.2": "us-east-1",
+        "s3.amazonaws.com": "aws-test"
+    },
+    (...)
+
+~/.aws/credentials
+^^^^^^^^^^^^^^^^^^
+
+.. TIP::
+   If you set the :code:`credentials` object in your
+   :code:`locationConfig.json` file, you may skip this section
```

Comment:
why can you skip?  because you set the accessKey/secretKey directly?  unclear.

Reply:
I think that, because `credentials` is an object when `credentialsProfile` is a string entry in the dict, the wording will avoid confusion (and it's documented two sections before that). I can specify `credentials.accessKey` and `credentials.secretKey` if you think that will be clearer.

Reply:
I think it would be clearer to say -- If you explicitly set your accessKey and secretKey in in the credentials section of your aws_s3 location, then you do not need a credentials file in your home directory and can skip the next step.